examples/linear_extension/linear_extension.jl

```diff
@@ -14,31 +14,31 @@ function run_linear_extension_example()
     # -------------------------------
     ## scalar parameters
     E = 2.0             # Young modulus in Pa
-    ν = 0.4             # Poisson's ratio
+    nu = 0.4             # Poisson's ratio
```

Comment:
`\nu` is probably fine here? or do you also experience issues with it?

Reply:
as discussed on friday. my proposal is to slowly remove Unicode from examples. particularly the introductory ones